Echo.Core/src/Textures/Colors/RGBA128.Parser.cs

```diff
@@ -141,9 +145,84 @@ bool ParseHex(out RGBA128 result)
 			static int CombineHex(int hex0, int hex1) => hex0 * 16 + hex1;
 		}
 
-		bool ParseRGB(out RGBA128 result) => throw new NotImplementedException();
+		bool ParseRGB(out RGBA128 result)
+		{
+			int index = 0;
+
+			int r = 0, g = 0, b = 0, a = 255;
+
+			if (findNextInt(content, ref index, ref r) &&
+				findNextInt(content, ref index, ref g) &&
+				findNextInt(content, ref index, ref b))
+			{
+				if (findNextInt(content, ref index, ref a)) return YieldError(out result);
+
+
+				if (IsChannelInRange(r) &&
+					IsChannelInRange(g) &&
+					IsChannelInRange(b) &&
+					IsChannelInRange(a))
+				{
+					result = new RGBA128(r / 255f, g / 255f, b / 255f, a / 255f);
+					return true;
+				}
+
+			}
+			return YieldError(out result);
+
+			static bool IsChannelInRange(int channel) => channel > 0 && channel <= 255;
+			static bool findNextInt(ReadOnlySpan<char> src, ref int index, ref int result)
+			{
+				int len = 0;
+
+				while (!char.IsDigit(src[index])) if (src.Length <= index++) return false; // find first digit
+				while (char.IsDigit(src[index + len]) && (index + len) < (src.Length - 1)) len++; // get the length of the number
+
+				if (len == 0) return false;
+
+				ReadOnlySpan<char> num = src.Slice(index, len);
+				result = int.Parse(num);
+				index += len;
+				return true;
+			}
+		}

```

Comment:
Small code style suggestions for this method:
- Method names should be `PascalCase` not `camelCase` (note the first uppercase)
- Do not use short abbreviations unless they make the lines _extremely_ long (ie. src -> source, len -> length)
- Do not put two one line statements into one line (ie. don't do `while (something) if (somethingElse) thenDoStuff`) because it gets confusing, add braces and put them on new lines. 
- It is probably better to rename `index` to `head` since we are scanning through a `ReadOnlySpan` and `index` is the 'head' of the scanning.
- It is probably better to record the end of the number rather than the length of it, since then you do not need to to `idnex + len` every time. And then after processing the current integer you can just do `index = end` (also renaming `index` to `head` and `end` to `tail` probably makes more sense, so record `int tail = head;` after your first while loop when you found the start of the integer, then increment the `ref head` to find the 'head' of the integer).
- Use `int.TryParse` rather than `int.Parse` because we do not want exceptions but rather a `bool` to report whether the conversion succeeded or not.
- `ReadOnlySpan` has range slicers, so you can do `int.TryParse(source[tail..head], out result)`.